### PR TITLE
Pf v df

### PR DESCRIPTION
**Pull request by fourndo**
Add notebook and rst file for PF tutorial (MAG)

Need to be on pf\dev to run

### CODE REVIEW

File: docs/content/stories/Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
```
Comment (lheagy) [inline docs/content/stories/Untitled.ipynb]: can you rename the notebook to "TKC_Mag.ipynb" ? or something along those lines?

Comment (lheagy) [inline docs/content/stories/Untitled.ipynb]: or if this is not used as a part of the tutorial, could you remove it?

**Comment by lheagy** [comment]
I have put the data and mesh files on google app engine: 
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/GravData.obs 
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/Initm.den 
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/Initm.sus
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/MagData.obs 
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/Mesh.msh
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/SimPEG_Grav_Input.inp
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/SimPEG_MAG_l2l2.sus 
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/SimPEG_MAG_lplq.sus 
- https://storage.googleapis.com/simpeg/tkc_synthetic/potential_fields/SimPEG_PF_Input.inp

These can be downloaded there instead of having them stored in the repo. Here is an example of downloading files using python. 
http://stackoverflow.com/questions/22676/how-do-i-download-a-file-over-http-using-python

Would you mind using the download call instead of storing the data / mesh files here? Let me know if you run into any issue (or if I forgot any files!)

**Comment by fourndo** [comment]
That will be interesting. Maybe I can add a _Remote_ switch in the Driver?

**Comment by fourndo** [comment]
That will be interesting. Maybe I can add a _Remote_ switch in the Driver?

**Pull request closed without merging**